facade/nodeFacade.go

```diff
@@ -358,9 +364,11 @@ func (nf *nodeFacade) GetAccounts(addresses []string, options apiData.AccountQue
 
 	response := make(map[string]*apiData.AccountResponse)
 	var blockInfo apiData.BlockInfo
+	ctx, cancel := nf.getContextForApiTrieRangeOperations()
+	defer cancel()
 
 	for _, address := range addresses {
-		accountResponse, blockInfoForAccount, err := nf.node.GetAccount(address, options)
+		accountResponse, blockInfoForAccount, err := nf.node.GetAccount(address, options, ctx)
```

Comment:
I think we have an old issue here

seems that here we could return accounts with different block info, but in the end we reply with the latest block info, which might not be correct.

Could you request with the same roothash for all accounts?